### PR TITLE
Implement showcase of companies using RxJS

### LINKED ISSUE #10: Create a companies page
[ ] Get approval for format of page
[ ] Design layout of page
[ ] Collect set of companies using RxJS
[ ] Create the ability for people to submit their company
[ ] Create the ability for approvals (is this just submitting a PR or should it be a form?
[ ] Finish creating the page!

### PR DESCRIPTION
**Pull request by ashwin-sureshkumar**
- Construct a list of companies which utilize RxJS
- Build a showcase for the list of companies

Close #10

### CODE REVIEW

File: src/app/companies/companies-list.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Company } from './companies.model';
+
+export const COMPANIES_LIST: Company[] = [
+  {
+    name: 'Google',
+    location: 'California',
+    logo:
+      'https://upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Google-favicon-2015.png/150px-Google-favicon-2015.png',
```
Comment (Airblader) [inline src/app/companies/companies-list.ts]: I think the logos should actually be checked into the repository. Failing that, they should be taken from the official sources, not wikimedia  or some blog.

File: src/app/companies/companies.component.html [hunk 2]
```diff
@@ -1 +1,21 @@
+<div fxLayout="row" fxLayoutAlign="space-evenly center" fxLayoutGap="10px" fxLayoutWrap class="companies-container">
+  <mat-card *ngFor="let company of companies | async" fxFlex.xs="100" fxFlex.sm="45" fxFlex.md="30" fxFlex.lg="23">
+    <mat-card-header>
+      <div mat-card-avatar>
+        <img [src]="company.logo" class="company-logo">
+      </div>
+      <mat-card-title>{{company.name}}</mat-card-title>
+      <mat-card-subtitle>
+        {{company.location}}
+        <button class="website-button" mat-icon-button aria-label="website" (click)="openWindow(company.website)">
```
Comment (JWO719) [inline src/app/companies/companies.component.html]: For better accessibility this should be an a - tag with a proper href

File: src/app/companies/companies-list.ts (hunk 1 above)
Comment (ashwin-sureshkumar) [reply src/app/companies/companies-list.ts]: @Airblader - This is a temporary for the PR to get approved, I forgot to mention that. This list should be updated with official list of companies.

File: src/app/companies/companies.component.html (hunk 2 above)
Comment (ashwin-sureshkumar) [reply src/app/companies/companies.component.html]: Ah, my bad. I was updating an existing PR and missed this. Will push up the changes.

**Comment by ashwin-sureshkumar** [comment]
@Airblader @JWO719 - Changes addressed, please re-review.

File: src/app/companies/companies.component.ts [hunk 3]
```diff
@@ -1,10 +1,23 @@
...
   styleUrls: ['./companies.component.scss']
 })
 export class CompaniesComponent {
-  constructor() {}
+  companies: Observable<Company[]>;
+  constructor(private companyService: CompanyService) {
+    this.companies = this.companyService.getCompanies();
+  }
+
+  openWindow(url: string) {
```
Comment (JWO719) [inline src/app/companies/companies.component.ts]: method isn't needed anymore, I think,

Comment (ladyleet) [reply src/app/companies/companies.component.ts]: @JWO719 really appreciate you taking the time to review PRs!

Comment (JWO719) [reply src/app/companies/companies.component.ts]: thank you. It's a pleasure

**Comment by ladyleet** [comment]
@ashwin-sureshkumar phew!

**Pull request merged**